components/concept/fetch/src/test/java/mozilla/components/concept/fetch/ClientTest.kt

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.fetch
+
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class ClientTest {
+    @Test
+    fun `Verify default user agent format`() {
+        val client = TestClient()
+
+        val userAgent = client.exposeDefaultHeaders().get("User-Agent")
+        assertNotNull(userAgent!!)
+
+        assertTrue(userAgent.matches("MozacFetch/[0-9]+\\.[0-9]+\\.[0-9]+".toRegex()))
+    }
+
+    @Test
+    fun `Expects gzip encoding default header`() {
+        val client = TestClient()
+
+        val acceptEncoding = client.exposeDefaultHeaders().get("Accept-Encoding")
+        assertNotNull(acceptEncoding!!)
+        assertEquals("gzip", acceptEncoding)
+    }
+
+    @Test
+    fun `Expects accept all default header`() {
+        val client = TestClient()
+
+        val accept = client.exposeDefaultHeaders().get("Accept")
+        assertNotNull(accept!!)
+        assertEquals("*/*", accept)
+    }
+
+    @Test
+    fun `Expects keep-alive connection default header`() {
+        val client = TestClient()
+
+        val connection = client.exposeDefaultHeaders().get("Connection")
+        assertNotNull(connection!!)
+        assertEquals("keep-alive", connection)
+    }
+
+    @Test
+    fun `Async request with coroutines`() = runBlocking {
```

Comment:
![image](https://user-images.githubusercontent.com/773158/48353686-092c1b80-e65e-11e8-9421-d3b4ff20239c.png)

Reply:
Should we also fix this lint issue ?